nimare/tests/test_annotate_gclda.py

```diff
@@ -7,12 +7,12 @@
 
 from nimare import annotate, decode
 
+# Disable numba execution to allow pytest-cov to detect coverage in functions with @jit
+config.DISABLE_JIT = True
+
 
 def test_gclda_symmetric(testdata_laird):
     """A smoke test for GCLDA with symmetric regions."""
-    # Disable numba execution to allow pytest-cov to detect coverage in functions with @jit
-    config.DISABLE_JIT = True
```

Comment:
Can you do something like `with config.DISABLE_JIT`?

Reply:
I tried running the GCLDA test with `with config.DISABLE_JIT`, and it didn't work.
It looks like disabling jit for GCLDA and enabling it again for the other tests that use sparse is working.

Reply:
The Codecov report still does not cover functions with `@jit`. 
The issue is that disabling numba globally produces an error in the functions that use sparse arrays.